images/Images/ImageInfo.h

```diff
@@ -289,7 +289,6 @@ class ImageInfo : public RecordTransformable
 
     // Check if the beam shape matches the coordinates.
     void checkBeamShape (uInt& nchan, uInt& npol,
-                         const ImageInfo& info,
                          const IPosition& shape,
                          const CoordinateSystem& csys) const;
 
```

Comment:
That's correct. I'll make it private and add an underscore.
It was a bit strange having an ImageInfo member function that checked an ImageInfo argument instead of the object itself.